tests/pipeline/nodes/model/mtcnnv1/test_mtcnn.yml

```diff
@@ -0,0 +1,31 @@
+input: ["img"]
+output: ["bboxes", "bbox_labels", "bbox_scores"]
+
```

Comment:
Is this the right file? Looks like it's for yolo

Reply:
Changed to mtcnn yml.